_bookdown.yml

```diff
@@ -0,0 +1 @@
+book_filename: ggplot2
```

Comment:
I don't normally bother with customising the file name, but this file should contain `new_session: yes`, and should list the `rmd_files` in order.